content/en/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: blog
+title: "Volume Health Monitoring goes Alpha again"
+date: 2021-04-16
+slug: volume-health-monitoring-go-alpha
+---
+
+**Authors:** Xing Yang (VMware)
+
+A second alpha version of CSI Volume Health Monitoring feature is being released with Kubernetes 1.21. It was introduced as an alpha feature in Kubernetes 1.19. In Kubernetes 1.21, a second alpha was done with some design changes.
```

Comment:
I recommend rewriting this to make the past tense clear - v1.21 is already released.

Reply:
Sure.

Reply:
Done